docs/restapi.rst

```diff
@@ -0,0 +1,7 @@
+REST API
+========
+
+The REANA Server offers a REST API for management of workflow running in REANA
```

Comment:
I am not sure if this would be `s/workflow/analyses`, or not specifying anything at all and just saying management of the REANA Cloud. Anyhow, not only workflows (understanding that analyses includes also single job/step tasks like https://github.com/reanahub/reana-demo-helloworld once we remove `yadage`)